tests/test_api.py

```diff
@@ -54,9 +54,11 @@ def test_get_variant_hashes_by_priority_roundtrip(mocker, configs):
 
     namespace_priorities = ["test_namespace", "second_namespace"]
 
-    combinations: list[VariantDescription] = list(
-        get_combinations(configs, namespace_priorities)
-    )
+    # The null-variant is always the last one and implicitly added
+    combinations: list[VariantDescription] = [
+        *list(get_combinations(configs, namespace_priorities)),
```

Comment:
Why not modify `get_combinations()` to include it?

Reply:
Ah sure 😅 I didn't think about it 👌